vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -56,37 +68,59 @@ function Get-FilesToCacheOnVHD
+            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.15.12/windowszip/v1.15.12-1int.zip",
+            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.15.12-hotfix.20200714/windowszip/v1.15.12-hotfix.20200714-1int.zip",
+            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.15.12-hotfix.20200623/windowszip/v1.15.12-hotfix.20200623-1int.zip",
+            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.16.10/windowszip/v1.16.10-1int.zip",
+            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.16.10-hotfix.20200623/windowszip/v1.16.10-hotfix.20200623-1int.zip",
+            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.16.11/windowszip/v1.16.11-1int.zip",
+            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.16.11-hotfix.20200617/windowszip/v1.16.11-hotfix.20200617-1int.zip",
+            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.16.12/windowszip/v1.16.12-1int.zip",
+            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.16.13/windowszip/v1.16.13-1int.zip",
+            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.16.13-hotfix.20200714/windowszip/v1.16.13-hotfix.20200714-1int.zip",
+            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.17.7/windowszip/v1.17.7-1int.zip",
+            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.17.7-hotfix.20200714/windowszip/v1.17.7-hotfix.20200714-1int.zip",
+            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.17.8/windowszip/v1.17.8-1int.zip",
+            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.17.9/windowszip/v1.17.9-1int.zip",
+            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.17.9-hotfix.20200714/windowszip/v1.17.9-hotfix.20200714-1int.zip",
+            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.18.4/windowszip/v1.18.4-1int.zip",
+            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.18.4-hotfix.20200624/windowszip/v1.18.4-hotfix.20200624-1int.zip",
+            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.18.5/windowszip/v1.18.5-1int.zip",
+            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.18.6/windowszip/v1.18.6-1int.zip",
+            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.19.0-rc.4/windowszip/v1.19.0-rc.4-1int.zip"
         );
         "c:\akse-cache\win-vnet-cni\" = @(
-            "https://acs-mirror.azureedge.net/azure-cni/v1.0.33/binaries/azure-vnet-cni-windows-amd64-v1.0.33.zip"
-            "https://acs-mirror.azureedge.net/azure-cni/v1.1.3/binaries/azure-vnet-cni-singletenancy-windows-amd64-v1.1.3.zip"
-            "https://acs-mirror.azureedge.net/azure-cni/v1.1.6/binaries/azure-vnet-cni-singletenancy-windows-amd64-v1.1.6.zip"
+            "https://kubernetesartifacts.azureedge.net/azure-cni/v1.1.2/binaries/azure-vnet-cni-singletenancy-windows-amd64-v1.1.2.zip",
```

Comment:
pls use acs-mirror as it's the allowed egress list, while kubernetesartifacts is not.

This will help test the availability of acs-mirror CDN as well

Reply:
From my understanding, cached file does not impact egress. These URLs are only used in building the Windows VHD.

Reply:
Yeah, agree with Abel, this is for vhd building only. is this necessary?

Reply:
True, but it's always safer to exercise on the same url ,  Having two different urls is just confusing, plus it's a good/free chance to test those acs-mirrors are really working.

Reply:
Here is the PR: https://github.com/Azure/AgentBaker/pull/230